src/sandbox/alternative-threading/CountDownLatch2.cs

```diff
@@ -60,7 +60,7 @@ public int Remaining
 		/// <summary>
 		/// Waits forever for the latch to be completed
 		/// <summary>
-        public bool await()
+        public bool Await()
```

Comment:
This was causing some issues because of the reserved keyword `await` (in the AsyncAwait TPL)